doc/source/user_guide/dsintro.rst

```diff
@@ -88,7 +88,8 @@ index will be pulled out.
 **From scalar value**
 
 If ``data`` is a scalar value, the value will be repeated to match
-the length of **index**.
+the length of **index**.  If the index is not provided, it defaults
+to ``RangeIndex (0, 1, 2, …, n)``.
```

Comment:
`n` here is not defined, and I think in this notation would be `0`, but this is not the notation used to construct a RangeIndex. In particular, `RangeIndex(0)` here is wrong. Can you make this `RangeIndex(1)`.

Reply:
I made the change.